src/System.Runtime/ref/System.Runtime.Manual.cs

```diff
@@ -16,29 +16,88 @@
 
 namespace System
 {
+    /// <summary>
+    /// Represents a single-precision floating-point number.
+    /// </summary>
     public partial struct Single
     {
+        /// <summary>
+        /// Represents the smallest possible value of <see cref="Single" />. This field is constant.
+        /// </summary>
         public const float MinValue = (float)-3.40282346638528859e+38;
+        /// <summary>
+        /// Represents the smallest positive <see cref="Single" /> value that is greater than
+        /// zero. This field is constant.
+        /// </summary>
         public const float Epsilon = (float)1.4e-45;
+        /// <summary>
+        /// Represents the largest possible value of <see cref="Single" />. This field is constant.
+        /// </summary>
         public const float MaxValue = (float)3.40282346638528859e+38;
+        /// <summary>
+        /// Represents positive infinity. This field is constant.
+        /// </summary>
         public const float PositiveInfinity = (float)1.0 / (float)0.0;
+        /// <summary>
+        /// Represents negative infinity. This field is constant.
+        /// </summary>
         public const float NegativeInfinity = (float)-1.0 / (float)0.0;
+        /// <summary>
+        /// Represents not a number (NaN). This field is constant.
+        /// </summary>
         public const float NaN = (float)0.0 / (float)0.0;
     }
+    /// <summary>
+    /// Represents a double-precision floating-point number.
+    /// </summary>
     public partial struct Double
     {
+        /// <summary>
+        /// Represents the smallest possible value of a <see cref="Double" />. This field is
+        /// constant.
```

Comment:
I've always found it a bit strange that we say things like "This field is constant".  While this is technically fine, I'm not sure most developers think of consts as being fields.